jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -2921,6 +2921,35 @@ public String toGenericString() {
 	if (result.length() > 0) {
 		result.append(' ');
 	}
+
+	/*[IF JAVA_SPEC_VERSION >= 23]*/
+	if (Modifier.isFinal(modifiers)) {
+		// A final class can be neither sealed nor non-sealed.
+	} else if (isSealed()) {
+		result.append("sealed ");
```

Comment:
Pls add the non-nls comments.

Reply:
Done in 35ce15226b391eb4c0099673ebb9634f5d33fe8d.